src/components/App.tsx

```diff
@@ -1,7 +1,7 @@
-import Rate from './Rate';
+import Rate from './Rate/Rate';
 import RegisteredMembers from './RegisteredMembers';
 import { Routes, Route } from "react-router-dom";
-import Rates from './Rates';
+import Rates from './Rates/Rates';
```

Comment:
why do you need folder for each and every component?

Reply:
Usually in React (how I learned in my school ) ) files of components grouped  in own directories with additional files like as constants, styles and etc. But I checked  React  docs and I think you are right in our case we have a not big app and this schema without own directories or file organization is good